plugins/catalog-backend/src/ingestion/processors/github/github.ts

```diff
@@ -203,7 +208,8 @@ export async function getOrganizationTeams(
     }
 
     const memberNames: string[] = [];
-    groupMemberUsers.set(team.slug, memberNames);
+    const groupKey = orgNamespace ? `${orgNamespace}/${team.slug}` : team.slug;
```

Comment:
Hm, yeah I see what you are doing here. It's a bit unfortunate that the key here is a string, since if you defined that your user is `memberOf: ['group:someNamespace/someName']` (i.e. over-specifying the ref to include the optional kind as well), this won't match the string `someNamespace/someName` that comes out of this.

Not a big deal though. Let's keep this and then consider refactoring to use a full `EntityName` always instead, or similar